src/Workspaces/CSharpTest/CodeGeneration/AddImportsTests.cs

```diff
@@ -875,11 +874,10 @@ class C1 {}
 
 class C
 {
-    global::B.C1 M(A.C2 c2) => default;
+    C1 M(A.C2 c2) => default;
```

Comment:
the prior behavior demonstrates why we wanted to make this change.  Features that generate code and want imports to be added should not cause existing code to now be qualified like how it was doing.  We simply should not add the import in this case and we should leave the users code alone.  

This will mean that the newly generated code may end up more qualified than before.  But that's fine.  It's new code, and the user can decide how best to simplify it.

Reply:
The before/after here is now the same, right? Is it worth extracting to a local to make the test intent easier?

Reply:
yup.  i added a new TestNoImportsAdded helper to make this clear.